web/templates/compose_banner/unmute_topic_banner.hbs

```diff
@@ -0,0 +1,7 @@
+{{#> compose_banner }}
+    <p>
+        {{#tr}}
+        Your message was sent to a {muted_narrow} you have muted. You will not receive notifications about new messages.
+        {{/tr}}
```

Comment:
Unfortunately the `muted_narrow` variable doesn't work with i18n, for two reasons: One is that it's itself not tagged for translation, and even if it was, in other languages, the rest of the sentence needs to be conjugated differently depending on things like the gender of the word for "stream".

So what we should do here is just have a normal if/else clause for which string to use here. (And we should move the second sentence into its own translation tag outside of that, so it only needs to be translated once).

Reply:
Understood, thanks. Have made the change!